config/peerpods/podvm/libvirt-podvm-image-cm.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: libvirt-podvm-image-cm
+  namespace: openshift-sandboxed-containers-operator
+data:
+  CLOUD_PROVIDER: "libvirt"
+  # PodVM image distro
+  PODVM_DISTRO: "rhel"
+
+  # Pod VM sources
+  # If changing the source, then ensure the respective payload binaries are available
+  # for the new source
+  CAA_SRC: "https://github.com/confidential-containers/cloud-api-adaptor"
+  CAA_REF: "v0.8.2"
+
+  # Booleans
+  INSTALL_PACKAGES: "no"
+  DOWNLOAD_SOURCES: "no"
+  CONFIDENTIAL_COMPUTE_ENABLED: "no"
+
+  # Livirt specific
+  ORG_ID: ""
+  ACTIVATION_KEY: ""
+  YQ_VERSION : "v4.35.1"
+  YQ_CHECKSUM: "sha256:4e6324d08630e7df733894a11830412a43703682d65a76f1fc925aac08268a45"
+  ARCH: "s390x"
+  GO_VERSION: "1.21.9"
```

Comment:
Is GO_VERSION needed here ?

Reply:
IIUC yq and go are used only in order build the tools right? (i.e. not peerpods/coco components) in that  case, i think it would be ok to use fixed versions in script and don't allow the user to set these.